kolibri/plugins/coach/assets/src/views/exams-page/create-exam-modal.vue

```diff
@@ -20,7 +20,7 @@
       <k-button
         :text="$tr('createExam')"
         :primary="true"
-        :disabled="!selectedChannel"
```

Comment:
It might be simpler to keep the original predicate for `disabled` and intialize `data.selectedChannel` to be something falsy like `null`. The problem is that the original initial value `{}` is truthy in Javascript.

```diff
    data() {
-      return { selectedChannel: {} };
+      return { selectedChannel: null };
    },

```

Reply:
Initializing selectChannel to null causes errors in the console.

Reply:
I see. The k-select component only allows Objects and interprets a `{}` option as not having selected anything.

Reply:
It might get closer to the intention behind the validation to check that `selectedChannel.value` is truthy, as it being undefined was what was probably causing errors before.

As unlikely as it is, if somehow `selectedValue` were `{ foo: 'bar' }`, the button would still be enabled, but lead to the same problem of `selectedChannel.value` being undefined.

Reply:
These are the outputs of the changes you want
![image](https://user-images.githubusercontent.com/7971405/36058216-b2421ab6-0dcf-11e8-8ea7-052ba546c301.png)

Reply:
I think the risk of `selectedValue` being `{ foo: 'bar' }` is very low, given how the code is constructed, no? I agree it would be better for k-select to accept `null`, and do stronger validation (because it doesn't just require an object, it requires an object with `value` and `label`, I believe), but that's a separate issue from the one being fixed here.

Reply:
I'm not sure how changing `:disabled=!selectedChannel.value` could cause all of those errors. I just tried the change on my copy, and I didn't see anything.

Reply:
Created a follow up issue detailing the broader concerns raised here: https://github.com/learningequality/kolibri/issues/3187